docs/components/Footer.js

```diff
@@ -4,40 +4,80 @@ import { Box, Flex, Icon, Link, Text } from 'gestalt';
 import trackButtonClick from './buttons/trackButtonClick.js';
 
 export default function Footer(): Node {
+  const links = [
```

Comment:
nit: static consts like this can be defined outside the component, simplifying and making the component more readable (and a perf micro-optimization, but that doesn't matter)